README.md

```diff
@@ -231,3 +231,15 @@ networks:
   - network: default
     nic_model: e1000
 ```
+### Development
+
+You can run a development copy in a virtual env:
+
+```shell
+python3 -mvenv ~/tmp/my-greate-venv
+source ~/tmp/my-greate-venv/bin/activate
+pip3 install -e /path/to/my-virt-lightning-copy
```

Comment:
AFAIR, you need pip3 install -U -e here to resolve possible conflict with VL installed system-wide

Reply:
Total unfamiliar with pip, but reading a little, -U,--upgrade forces package upgrades, --user configures alternative install locations, which seems reasonable but it seems pip doesn't support --user in default venv setups

https://pip.pypa.io/en/stable/cli/pip_install/
https://pip.pypa.io/en/stable/user_guide/

I'll try a little testing, but so far i haven't noticed any conflict.